src/api/api.js

```diff
@@ -80,3 +80,12 @@ export function getUsers() {
       return err;
     });
 }
+
+export function deleteCommentByCommentId(comment_id){
+  return myApi.delete(`/comments/${comment_id}`).then((response)=>{
+    return response
+  }).catch((err)=>{
```

Comment:
You could just have the one catch block in the component that is calling the function to make this a bit easier and cleaner, then just catch the error and handle it however you like while in the right place to do so.